.github/workflows/build.yaml

```diff
@@ -306,6 +306,7 @@ jobs:
           mkdir -p ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
           helm package ${{ env.HELM_CHARTS_DIR }} --version ${{ env.CHART_VERSION }} --app-version ${{ env.REL_VERSION }} --destination ${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
       # TODO: Delete this step once we use GHCR as the helm chart repo.
+      # Cannot use Workload Identity because azure federated identity doesn't accept wildcard tag version.
```

Comment:
I cannot remove this code because we still use ACR for helm chart. I think we need to move helm chart to ghcr as soon as possible.  cc/ @willdavsmith  @rynowak  @AaronCrawfis